src/flow-bit.c

```diff
@@ -138,69 +138,52 @@ void FlowBitFree(FlowBit *fb)
 #ifdef UNITTESTS
 static int FlowBitTest01 (void)
 {
-    int ret = 0;
-
     Flow f;
     memset(&f, 0, sizeof(Flow));
 
     FlowBitAdd(&f, 0);
 
     FlowBit *fb = FlowBitGet(&f,0);
-    if (fb != NULL)
-        ret = 1;
-
     GenericVarFree(f.flowvar);
-    return ret;
+    FAIL_IF_NULL(fb);
+    
+    PASS;
 }
 
 static int FlowBitTest02 (void)
 {
-    int ret = 0;
-
     Flow f;
     memset(&f, 0, sizeof(Flow));
 
     FlowBit *fb = FlowBitGet(&f,0);
-    if (fb == NULL)
-        ret = 1;
-
     GenericVarFree(f.flowvar);
-    return ret;
+    FAIL_IF_NOT_NULL(fb);
+
+    PASS;
 }
 
 static int FlowBitTest03 (void)
 {
-    int ret = 0;
-
     Flow f;
     memset(&f, 0, sizeof(Flow));
 
     FlowBitAdd(&f, 0);
 
     FlowBit *fb = FlowBitGet(&f,0);
-    if (fb == NULL) {
-        printf("fb == NULL although it was just added: ");
-        goto end;
```

Comment:
Do you think it's better to just edit `end` with `FAIL;` so I don't have to add the `GenericVarFree(f.flowvar);` after every fail scenario?

Reply:
By adding the FAIL macro here, we get a better line number to check on failure.

Reply:
@jasonish are you suggesting that I should just use the FAIL macro in the if statements instead of using the FAIL_IF_NULL and FAIL_IF_NOT_NULL macros?

Reply:
No.  A FAIL_IF_NULL(fb) is ideal here.

Reply:
Understood, thanks! I've created a new PR with the suggested edit.